packages/sku/package.json

```diff
@@ -2,32 +2,31 @@
   "name": "sku",
   "version": "15.4.0",
   "description": "Front-end development toolkit, powered by Webpack, Babel, Vanilla Extract and Jest",
-  "types": "./dist/index.d.ts",
+  "types": "./dist/index.d.mts",
   "bin": {
     "sku": "bin/bin.js"
   },
   "exports": {
-    ".": {
-      "types": {
-        "import": "./dist/index.d.ts",
-        "require": "./dist/index.d.cts"
-      }
-    },
-    "./jest-preset": "./dist/config/jest/preset.js",
-    "./config/eslint": {
-      "types": "./dist/config/eslint/config.d.ts",
-      "default": "./dist/config/eslint/config.js"
-    },
-    "./config/storybook": {
-      "types": "./dist/config/storybook/config.d.ts",
-      "default": "./dist/config/storybook/config.cjs"
-    },
-    "./vite/client": {
-      "types": "./dist/services/vite/client.d.ts"
-    },
-    "./@loadable/component": "./src/@loadable/component/index.ts",
-    "./webpack-plugin": "./dist/services/webpack/config/plugins/skuWebpackPlugin.js",
-    "./package.json": "./package.json"
+    ".": "./dist/index.mjs",
```

Comment:
I've become somewhat apprehensive towards making all of these public exports. It's effectively making them public, yet undocumented, API. I feel like it's only a matter of time before someone starts consuming an endpoint that was never designed to be consumed publicly, and we accidentally break them usage.

Would these work as [imports](https://nodejs.org/api/packages.html#subpath-imports) instead?